src/EnergyPlus/DataSurfaces.hh

```diff
@@ -823,6 +860,95 @@ namespace DataSurfaces {
         }
     };
 
+    enum class NfrcProductOptions : int
+    {
+        Invalid = -1,
+        CasementDouble,
+        CasementSingle,
+        DualAction,
+        Fixed,
+        Garage,
+        Greenhouse,
+        HingedEscape,
+        HorizontalSlider,
+        Jal,
+        Pivoted,
+        ProjectingSingle,
+        ProjectingDual,
+        DoorSidelite,
+        Skylight,
+        SlidingPatioDoor,
+        CurtainWall,
+        SpandrelPanel,
+        SideHingedDoor,
+        DoorTransom,
+        TropicalAwning,
+        TubularDaylightingDevice,
+        VerticalSlider,
+        Num
+    };
+
+    constexpr std::array<std::string_view, static_cast<int>(NfrcProductOptions::Num)> NfrcProductName = {
```

Comment:
Same. Also, `Names` (not `Name`).

Reply:
I think the current pattern for input enums is to also declare `NfrcProductNamesUC` because all input alphas get pushed to UPPERcase, then `SameString` isn't required later.